modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/dto/v1_0/converter/ObjectEntryDTOConverter.java

```diff
@@ -222,40 +219,43 @@ public ObjectEntry toDTO(DTOConverterContext dtoConverterContext)
 	@Override
 	public ObjectEntry toDTO(
 			DTOConverterContext dtoConverterContext,
-			com.liferay.object.model.ObjectEntry objectEntry)
+			com.liferay.object.model.ObjectEntry serviceObjectEntry)
```

Comment:
Rename it to `serviceBuilderObjectEntry`. I couldn't find an example using only `service` as a prefix.

Reply:
Done [here](https://github.com/liferay-bpm/liferay-portal/pull/4271/commits/edd2c482f6f2bb1a14334b6bb8ed64c552b9e6ab)